Spatial_searching/include/CGAL/Fuzzy_iso_box.h

```diff
@@ -117,7 +117,7 @@ namespace CGAL {
 	  Cartesian_const_iterator_d pit = construct_it(p);
 	  Cartesian_const_iterator_d minit= min_begin, maxit = max_begin; 
 		for (unsigned int i = 0; i < dim; ++i, ++pit, ++minit, ++maxit) {
-			if ( ((*pit) < (*minit)) || ((*pit) > (*maxit)) ) return false;
+			if ( ((*pit) < (*minit)) || ((*pit) >= (*maxit)) ) return false;
```

Comment:
So the box is half-open (closed on the min side, open on the max side). Is there a particular reason for that choice?